src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -74,6 +74,7 @@
 use Symfony\Component\Lock\Store\FlockStore;
 use Symfony\Component\Lock\Store\StoreFactory;
 use Symfony\Component\Lock\StoreInterface;
+use Symfony\Component\Mailer\Transport;
```

Comment:
`use Symfony\Component\Mailer\Transport` not used here. Must be `use Symfony\Component\Mailer`?

Reply:
fixed